pkg/sysprom/initialize.go

```diff
@@ -614,7 +639,8 @@ func getVolumeMounts(dashboards []string) []apiv1.VolumeMount {
 		},
 	}
 
-	for i, d := range dashboards {
+	index := 0
```

Comment:
Not sure why we're using an external loop index here instead of the indice given by range. I didn't see a case where we skip an iteration which would make the index loop useful. Can you elaborate?

Reply:
Each dashboard is mounted as a configmap at a specific location, but cluster explorer is mounted in a different location to make it the default one. Due to this, there was one location here which was left empty and grafana was logging error messages, no valid dashboard found at this location. This was done to fix it.